docs/source/release_notes.rst

```diff
@@ -69,6 +70,8 @@ Breaking Changes
 * ``last_time_index``, ``secondary_time_index``, and ``interesting_values`` are no longer attributes
   of an entityset’s tables that can be accessed directly. Now they must be accessed through the metadata
   of the Woodwork DataFrame, which is a dictionary.
+* The helper function ``list_variable_types`` will be removed in a future release and replaced by ``list_logical_types``.
```

Comment:
We should add a section to `What's New in this Release` that describes the new list logical types and list semantic tags